d3a_api_client/setups/test_create_aggregator.py

```diff
@@ -53,8 +56,11 @@ def on_finish(self, finish_info):
         self.is_finished = True
 
 
-import os
-simulation_id = os.environ["API_CLIENT_SIMULATION_ID"]
+with open(os.environ['JSON_FILE_PATH']) as json_file:
+    config_info = json.load(json_file)
+
+
+simulation_id = config_info['uuid']
 domain_name = os.environ["API_CLIENT_DOMAIN_NAME"]
 websocket_domain_name = os.environ["API_CLIENT_WEBSOCKET_DOMAIN_NAME"]
```

Comment:
I think the goal is here to make the life easier for the user, so i suggest to handle this inside the `Aggregator` class and make the simulation_id, domain_name, websocket_domain_name optional parameters. If there are not set, the JSON file is read.

Reply:
I see the user would be reading this json file containing relevant simulation-info as they themselves have to accordingly get their script connected for their approved area

Reply:
Sorry, I thought this is the file that the user downloads from the d3a-web.
And it was designed so the user does not have to set all these parameters him/herself but only to read from this JSON file. So AFAIU, you need to read more parameters from there.

Reply:
I also understood from the task description that this information (like the websocket name and domain name) would be provided inside of the JSON file. So that information should probably overwrite the one contained in the `os.environ` dictionary.

Reply:
Yes, it is even now possible. I will leave it in this as an example.